lib/readline.js

```diff
@@ -170,6 +173,14 @@ function Interface(input, output, completer, terminal) {
     throw new ERR_INVALID_ARG_VALUE('completer', completer);
   }
 
+  if (history === undefined) {
+    history = [];
+  }
+
+  if (!(history instanceof Array)) {
+    throw new ERR_INVALID_ARG_VALUE('history', history);
+  }
```

Comment:
Using `else` to avoid checking the default value (which you already know is an array). Also you can use `validateArray` from `require('internal/validators')`
```suggestion
  if (history === undefined) {
    history = [];
  } else {
    validateArray(history, 'history');
  }
```

Reply:
Done